dev_guide/builds.adoc

```diff
@@ -1048,14 +1074,7 @@ it to the builder service account, and then your `BuildConfig`.
 $ oc secrets new mysecret .gitconfig=path/to/.gitconfig
 ----
 ====
-.. To create a secret from a `CA certificate`:
-+
-====
-----
-$ oc secrets new mysecret ca.crt=path/to/certificate
-----
-====
-.. To create a secret from a `CA certificate` and `.gitconfig`:
+.. To create a secret that combines a `.gitconfig` file and `CA certificate`:
```

Comment:
Strike this section (combining gitconfig and ca cert) and also clean up the basicauth section to only talk about providing a username and a password (not providing a ca cert or gitconfig).  Same cleanup in the ssh-auth section (don't discuss providing a gitconfig in that section).


Then create a new section "combining secrets" that discusses how to combing any permutation of  ssh-auth, basicauth info, ca.cert, and gitconfig into a single secret.